backend/main.py

```
@@ -11,208 +20,234 @@
+        channel.basic_publish(
+            exchange='',
+            routing_key='glaucoma_queue',
+            body=image_b64
+        )
+        connection.close()
+
+        return {
+            "status": "success", 
+            "message": "Image successfully sent to the queue! Waiting for the worker."
+        }
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)